### DISCUSSION TITLE
Repository integrity breach

### DISCUSSION THREAD

**Discussion by konon0v** [community/community#139161] [Repositories]
### Select Topic Area

General

### Body

My repository commits integrity was breached by some mailicious actor:
https://github.com/konon0v/superfluid-infra-private
This like it should look like:
![photo_2024-09-19 19 04 16](https://github.com/user-attachments/assets/98f43a0b-5d47-4cde-80d7-c7522b2b55e3)

![photo_2024-09-19 19 04 15](https://github.com/user-attachments/assets/b7bfcd19-2648-420a-8e9a-4805800a9b01)
This like it used to look like:
![photo_2024-09-19 19 04 14](https://github.com/user-attachments/assets/e7b9e7f5-fbf7-467f-8a65-64ec18f9ae10)

**Comment by ibra-kdbra** [comment]
contact support now, and change you repository name it may help.
